### PR TITLE
Remove python 2.7 support

### PR DESCRIPTION
**Pull request by jneves**
## Description
So, as we retired python 2.7 by accident on 0.49.0 we might as well finish the job. This removes all support for 2.7.

## GitHub Issues
#1994

### CODE REVIEW

File: tests/tests.py [hunk 1]
```diff
@@ -188,20 +175,16 @@ def test_get_manylinux_python38(self):
...
-        self.assertTrue(z.have_any_lambda_package_version('mysql-python'))
+        self.assertTrue(z.have_any_lambda_package_version('mysqlclient'))
         self.assertFalse(z.have_any_lambda_package_version('no_package'))
 
+        self.assertTrue(z.have_correct_lambda_package_version('mysqlclient', '1.3.12'))
+        self.assertFalse(z.have_correct_lambda_package_version('mysqlclient', '1.4.12'))
+
     def test_getting_installed_packages(self, *args):
-        z = Zappa(runtime='python2.7')
+        z = Zappa(runtime='python2.6')
```
Comment (mbeacom) [inline tests/tests.py]: Should this be `3.6` ?

File: requirements.txt [hunk 2]
```diff
@@ -2,7 +2,6 @@ argcomplete>=1.9.3
 boto3>=1.4.7
 durationpy>=0.5
 future>=0.16.0
```
Comment (mbeacom) [inline requirements.txt]: We should be able to remove `future` as well, no?

File: tests/tests.py (hunk 1 above)
Comment (jneves) [reply tests/tests.py]: Oops, fixed.

File: requirements.txt (hunk 2 above)
Comment (jneves) [reply requirements.txt]: Yes, changed that in #1998 though.

Comment (jneves) [reply requirements.txt]: Reverted on #1998 - both future and six will require more careful reviews for removal.

Comment (mathom) [reply requirements.txt]: I'd definitely like to clean up the imports to get rid of `future/past` cludge but I feel like that might be a bit further down the road. If we broke 2.7 already, just getting rid of it officially in the simplest manner makes the most sense to me.

**Pull request merged**